packages/theme/components/ValidationProviders/EmailValidationProvider.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <ValidationProvider
+    v-slot="{ errors }"
+    :rules="mergedRules"
+  >
+    <slot
+      :errors="errors"
+    />
+  </ValidationProvider>
+</template>
+
+<script>
+import { ValidationProvider } from 'vee-validate';
+import { computed } from '@nuxtjs/composition-api';
+
+export default {
+  name: 'EmailValidationProvider',
+  props: {
+    name: {
+      type: String,
+      default() {
+        return 'email';
+      }
+    },
+    rules: {
+      type: String,
+      default() {
+        return 'email';
+      }
+    },
+    'append-rules': {
+      type: String,
+      default() {
+        return '';
+      }
+    }
+  },
+  components: {
```

Comment:
Let's keep `components` right below the `name` property to stay consistent across other component definitions.